### ISSUE TITLE
RTL errors if the admin.macaroon is not found

### ISSUE THREAD

**Issue by andyhowell8** [ShahanaFarooqui/RTL#167]
**Describe the bug**
Running the 0.4.2 version of RTL throws a file not found exception for the `admin.macaroon` if it does not exist.

This was  previously fixed [#72](https://github.com/ShahanaFarooqui/RTL/issues/72), but has issue has been reintroduced.  

**To Reproduce**
Running the Docker container for 0.4.2
`docker run shahanafarooqui/rtl:0.4.2`

**Expected behavior**
RTL should handle exception better if possible. Version 0.4.1 does not have this issue.

**Actual behavior**
Stacktrace trying to read `admin.macaroon`

```
Status: Downloaded newer image for shahanafarooqui/rtl:0.4.2
fs.js:114
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/root/.lnd/data/chain/bitcoin/mainnet/admin.macaroon'
    at Object.openSync (fs.js:443:3)
    at Object.readFileSync (fs.js:343:35)
    at Object.connect.getAllNodeAllChannelBackup (/RTL/connect.js:439:44)
    at common.nodes.map.node (/RTL/connect.js:526:38)
    at Array.map (<anonymous>)
    at Object.connect.setServerConfiguration (/RTL/connect.js:526:16)
    at Object.<anonymous> (/RTL/rtl.js:5:36)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

**Your environment**
Just RTL docker container 

**Additional context**
Was not an issue in 0.4.1 seems to be a reintroduction of [#72](https://github.com/ShahanaFarooqui/RTL/issues/72)

**Comment by ShahanaFarooqui** [comment]
@andyhowell8, thanks for raising the issue. It is the side effect of issue #160 , we will fix and release next version on this weekend.

**Comment by ShahanaFarooqui** [comment]
@andyhowell8 The issue has been fixed and released in v0.4.3. Please check and let us know if you till face any issue. Closing the issue for now.

**Comment by andyhowell8** [comment]
Fantastic I'll give it a go tomorrow.
